react/features/conference/components/native/Conference.js

```diff
@@ -561,4 +565,21 @@ function _mapStateToProps(state) {
     };
 }
 
-export default withSafeAreaInsets(connect(_mapStateToProps)(Conference));
+export default withSafeAreaInsets(connect(_mapStateToProps)(props => {
+    const isConferenceScreenFocused = useIsFocused();
+
+    useEffect(() => {
+
+        if (isConferenceScreenFocused) {
+            setPictureInPictureEnabled(true);
+        } else {
+            setPictureInPictureEnabled(false);
+        }
+    });
```

Comment:
Wouldn't we want to filter this by `[ isConferenceScreenFocused `] ? At which point, can we rename it to "isFocused" ?

Reply:
From what I tested it's the same with or without input params. So I guess focus is by default checked.

Reply:
I was wrong about this, because we need to track when the component updates base on its visibility.